assets/css/css/style.css

```diff
@@ -0,0 +1,178 @@
+    background-size: cover;
+    font-family: 'Roboto Mono', monospace;
+    font-family: roboto;
+    color: white;
+}
+
+h3 {
+    color: #38f9d7;
+    text-transform: uppercase;
+    font-family: 'Roboto Mono 900', monospace;
+    font-weight: bolder;
+
+
+}
+h2 {
+    font-family: 'Roboto Mono 700', monospace;
+    font-size: 20px;
+    background: -webkit-linear-gradient(147deg, #FFE53B 0%, #FF2525 74%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+h6 {
+    font-family: 'Roboto Mono', monospace;
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    background: -webkit-linear-gradient(147deg, #FFE53B 0%, #FF2525 74%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+p {
+    font-size: 14px;
+    font-family: 'Roboto Mono', monospace;
+    text-align: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    background: -webkit-linear-gradient(147deg, #FFE53B 0%, #FF2525 74%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+}
+
+.jumbotron {
+    background-color: transparent;
+    margin-top: 10%;
+}
+
+#start-btn {
+    font-family: 'Roboto-Mono', monospace;
+    padding: 0.45em 1.5em;
+    border-radius: 3px;
+    background-color: transparent;
+    display: block;
+    text-align: center;
+    color: #FFC837;
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 25px;
+    align-content: center;
+    cursor: pointer;
+    border: 1px solid #FFC837;
+}
+
+#start-btn:hover {
+    color: #ffffff;
+    background-color: #FF8008;
+}
+
+.rounded {
+    border-radius: .5rem !important;
+}
+
+.container {
+    background: radial-gradient(circle, #9011ca 0%, #13364e 100%);
+    box-shadow: 0 0 15px rgb(46, 45, 45);
+    opacity: .8;
+    height: 100%;
+    text-align: center;
+    display: none;
+    margin-bottom: 5%;
+    padding-bottom: 2%;
+    margin-top: 5%;
+    border-radius: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
+    z-index: 0 !important;
+}
+
+#header {
+    margin-top: 20px !important;
+    z-index: 99 !important;
+    color: white;
+}
+.questions {
+    font-family: 'Roboto Mono 700', monospace;
+    font-size: 20px;
+    color:white;
+}
+
+label {
+    font-family: 'Roboto Mono 700', monospace;
+    padding: 2%;
+    font-weight: 900;
+    background: -webkit-linear-gradient(0deg, #43e97b 0%, #38f9d7 100%);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    font-size: 17px !important;
+}
+
+#sub-btn {
+    padding: 0.45em 1.5em;
+    border-radius: 3px;
+    background-color: transparent;
+    display: block;
+    text-align: center;
+    color: #38f9d7;
+    margin-right: auto;
+    margin-left: auto;
+    margin-top: 25px;
+    align-content: center;
+    cursor: pointer;
+    border: 1px solid #38f9d7;
+}
+
+#sub-btn:hover {
+    color: #ffffff;
+    background-color: #38f9d7;
+}
+
+.remainTime {
```

Comment:
I couldn't locate this elements, are this rules relevant to our project?

Reply:
This css was an old file I took to cut and paste over the new stylesheet, I removed it and only kept the one we are using.